nand2tetris/web-ide
# Feat: pwa manifest

Update manifest.json and generally make the PWA work.
Add service workers to load all files ahead of time.

To test this:

1. Run `npm run -w web serve-pwa` (requires installing latest node-modules, and having python3 available on your system path)
2. Visit localhost:8000/web-ide using Chrome or Edge
3. Click the "Install App" button
4. Test the app. Especially open web inspector, set Network to Offline, and ensure the app still functions as expected.
